### DISCUSSION TITLE
Connection timed out on oss.sonatype.org

### DISCUSSION THREAD

**Discussion by straumat** [community/community#26075] [Actions]
<p>My project <a href="https://github.com/cassandre-tech/cassandre-trading-bot" rel="nofollow noopener">Cassandre</a> is using actions and is publishing artifacts to <a href="https://oss.sonatype.org/content/repositories/snapshots" rel="nofollow noopener">https://oss.sonatype.org/content/repositories/snapshots</a>. Everything was working perfectly but since some days, all ci runs are failing with this kind of error :</p>
<pre><code class="lang-auto">[ERROR] Failed to execute goal org.apache.maven.plugins:maven-deploy-plugin:2.8.2:deploy (default-deploy) on project ***-trading-bot-spring-boot-autoconfigure: Failed to retrieve remote metadata tech.***.trading.bot:***-trading-bot-spring-boot-autoconfigure:2.0.1-SNAPSHOT/maven-metadata.xml: Could not transfer metadata tech.***.trading.bot:***-trading-bot-spring-boot-autoconfigure:2.0.1-SNAPSHOT/maven-metadata.xml from/to ossrh (https://oss.sonatype.org/content/repositories/snapshots): Transfer failed for https://oss.sonatype.org/content/repositories/snapshots/tech/***/trading/bot/***-trading-bot-spring-boot-autoconfigure/2.0.1-SNAPSHOT/maven-metadata.xml: Connection timed out (Read failed) -&gt; [Help 1]
</code></pre>
<p>I even relaunch previously working buijd and they know fail</p>
<p>Does anyone can help ?</p>

**Comment by weide-zhou** [comment]
<p>Hi <a class="mention" href="/straumat">@straumat</a>,</p>
<p>Glad to see you in Github Community Forum!</p>
<p>I notice your recent workflow is successful now after updating <code>maven-jar-plugin</code> from 3.1.0 to 3.2.0.<br>
Could you please confirm whether it’s fixed?</p>
<p>Thanks</p>

**Comment by straumat** [comment]
<p>Hi <a class="mention" href="/weide-zhou">@weide-zhou</a>, seems dependabot created a merge request and things are now fixed.</p>

**Comment by weide-zhou** [comment]
<p>Thanks <a class="mention" href="/straumat">@straumat</a> for your confirmation.</p>
<p>Let me know if you have any other questions.</p>
<p>Happy coding!</p>

**Comment by straumat** [comment]
<p><a class="mention" href="/weide-zhou">@weide-zhou</a> seems the problem is back :frowning:<br>
<a href="https://github.com/cassandre-tech/cassandre-trading-bot/actions/runs/212714620" rel="nofollow noopener">https://github.com/cassandre-tech/cassandre-trading-bot/actions/runs/212714620</a></p>
<p>Do you think it can come from my way to push updates now ?<br>
Dependabot seems to be usccesfull</p>

**Comment by weide-zhou** [comment]
<p>Hi <a class="mention" href="/straumat">@straumat</a>,</p>
<p>Can you revert the change to successful workflow and try again?<br>
You can add the change again for testing, i don’t have the secrets, sorry i cannot check on my side.<br>
But it mostly like a code issue while not related to Github Action.</p>
<p>Thanks</p>

**Comment by straumat** [comment, marked as answer]
<p>It works. Thanks seems the gitglow-maven plugin creates a problem when you use the feature action.</p>
